src/pages/index.js

```diff
@@ -72,12 +75,15 @@ const IndexPage = ({ books, posts }) => {
           </SimpleGrid>
 
           <Spacer />
+
           <Overline id="more">More API goodness</Overline>
           <SimpleGrid minChildWidth="300px" spacing={8} mt={4}>
             <PodcastFeed dark height="500" />
             {otherPosts.map((post) => {
               return <BlogPostItem key={post.slug} post={post} />;
             })}
+            <CarbonAd />
+            <Link href="/blog">Read more articles</Link>.
```

Comment:
Seems silly, but next/link expects an `<a>` tag to be within `<Link>` - https://nextjs.org/docs/api-reference/next/link

```suggestion
            <Link href="/blog">
              <a>
                Read more articles
              </a>
            </Link>.
```

Reply:
I copied a link from somewhere else to improve styling, check how it looks now.